src/dashboard/src/components/filter/Filter.tsx

```diff
@@ -14,14 +14,15 @@ export const Filter: React.FC = () => {
   return (
     <div className={styles.filter}>
       <h1>Filter</h1>
-      <Select options={tenantOptions} label="Tenant" placeholder="Select tenant" />
+      <Select className={styles.dropdown} options={tenantOptions} label="Tenant" placeholder="Select tenant" />
```

Comment:
These should be styled by the component itself as much as possible.  We don't need to apply it here.  Remove `className={styles.dropdown}` as it'll already be part of the component.

Reply:
removed className